CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+[0.0.6] - 2022-0X-XX
```

Comment:
Add target date!

Reply:
There's a date in my copy locally but git keeps insisting there is nothing to push.....